src/ResourceManager/DataLakeStore/Commands.DataLakeStore/DataPlaneModels/DataLakeStoreFileSystemClient.cs

```diff
@@ -495,28 +524,22 @@ public void AppendToFile(string filePath, string accountName, byte[] contents)
             }
         }
         /// <summary>
-        /// Setsup Nlog logging
+        /// Setsup Nlog logging to a file
         /// </summary>
         /// <param name="level">Logging level- debug or error</param>
         /// <param name="fileName">Path of file where logging will be done</param>
-        public void SetupLogging(LogLevel level, string fileName)
+        public void SetupFileLogging(LogLevel level, string fileName)
         {
-            // Step 1. Create configuration object 
-            var config = new LoggingConfiguration();
-
-            // Step 2. Create targets and add them to the configuration 
             var fileTarget = new FileTarget();
-            config.AddTarget("file", fileTarget);
+            _adlsLoggerConfig.AddTarget("file", fileTarget);
 
-            // Step 3. Set target properties 
             fileTarget.FileName = fileName;
-            fileTarget.Layout = "${message}";
+            
+            var rule = new LoggingRule("adls.dotnet.*", NLog.LogLevel.Debug, fileTarget);
+            _adlsLoggerConfig.LoggingRules.Add(rule);
```

Comment:
Can you describe what this actually logs?  We want to make sure that request method and headers and response status and headers are included

Reply:
@markcowl This is the debug line NLog posts:

2018-02-16 09:23:59.1720|DEBUG|adls.dotnet.WebTransport|HTTPRequest,Succeeded,cReqId:cc6758eb-656f-4bd7-b589-67165664e899.0,lat:6496,err,Reqlen:0,Resplen:0,token_ns:2,sReqId:2551abc6-ed04-4a99-b71f-264da768cf7d,path:/NewFile,qp:op=CREATE&overwrite=True&leaseid=30038d32-172f-4054-ac08-359421c24282&filesessionid=30038d32-172f-4054-ac08-359421c24282&CreateParent=True&write=true&syncFlag=DATA&api-version=2017-08-01

Reply:
@markcowl For failure case only we have the httpstatus in response like here we have InternalServorError:
2018-02-14 20:02:48.5660|DEBUG|adls.dotnet.WebTransport|HTTPRequest,failed,cReqId:e1dc5c6d-a9c1-4202-8d99-7e6c00d19f8d.0,lat:723,errInternalServerError: RuntimeException,Reqlen:15,Resplen:0,token_ns:0,sReqId:a2ed5f89-f8a7-42e6-af1c-187806135668,path:/Test/dir1/testConcurrentAppendParallel_15,qp:op=CONCURRENTAPPEND&appendMode=autocreate&api-version=2017-08-01